src/jdk.compiler/share/classes/com/sun/tools/javac/resources/javac.properties

```diff
@@ -292,6 +292,10 @@ javac.opt.X=\
     Print help on extra options
 javac.opt.help=\
     Print this help message
+javac.opt.help.lint=\
+    Print the supported keys of option -Xlint
+javac.opt.help.lint.header=\
+    The supported keys of option -Xlint are:
```

Comment:
Suggest: `The supported keys for -Xlint are:`
This does not affect the CSR.

Reply:
Fixed.